File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main (void) {

    int x, height, row;

    for (height = 0; height < 1 || height > 8; ) height = get_int ("Height: ");


    for (row = 0; row < height; row++) {

      for (x = 0; x < height - row - 1; x++) printf (" ");
      for (x = 0; x <= row ; x++) printf ("#");

      printf ("  ");

      for (x = 0; x <= row ; x++) printf ("#");
      //for (x = 0; x < height - row - 1; x++) printf (" ");

      printf ("\n");
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    n = get_int ("Height: ")
    if n > 0 and n <= 8:
        break

x = 1
y = n - 1

while x <= n:
    print (" " * y + "#" * x + "  " + "#" * x)
    x += 1
    y -= 1
